src/main/java/solvd/laba/ermakovich/hf/kafka/KafkaMessageMapper.java

```diff
@@ -0,0 +1,45 @@
+package solvd.laba.ermakovich.hf.kafka;
+
+import java.time.LocalDateTime;
+import java.util.UUID;
+import lombok.SneakyThrows;
+import org.springframework.stereotype.Component;
+import solvd.laba.ermakovich.hf.domain.Account;
+import solvd.laba.ermakovich.hf.domain.exception.IllegalOperationException;
+import solvd.laba.ermakovich.hf.event.account.CreateAccount;
+import solvd.laba.ermakovich.hf.event.account.DeleteAccount;
+import solvd.laba.ermakovich.hf.event.EventRoot;
+import solvd.laba.ermakovich.hf.event.IntegrationEvent;
+
+/**
+ * @author Ermakovich Kseniya
+ */
+@Component
+public class KafkaMessageMapper implements MessageMapper<EventRoot, IntegrationEvent> {
+
+    @SneakyThrows
+    public EventRoot toEvent(IntegrationEvent integrationEvent) {
+        return switch (integrationEvent.getEventType()) {
+            case CreateAccount.EVENT_TYPE -> {
+                Account account = Account.builder()
+                        .userId(UUID.fromString(integrationEvent.getPayload()))
+                        .build();
+                yield CreateAccount.builder()
```

Comment:
what is the `yield`?

Reply:
it's need to return result from switch branch. Here I can't use return